solax/inverters/x3_hybrid_g4.py

```diff
@@ -129,7 +129,7 @@ def response_decoder(cls):
                 Measurement(Units.KWH, storage=True),
                 div10,
             ),
-            "Battery mode": (168, Units.NONE),
+            # "Battery mode": (168, Units.NONE), # Only use the index once due to HA uids
             "Battery mode text": (168, Units.NONE, X3HybridG4._decode_battery_mode),
```

Comment:
Could you please change the remaining sensor to 'Battery mode' instead of 'Battery mode text'? Then i will be happy to merge this.